db-script/db.sql

```diff
@@ -1,30 +1,27 @@
-
 CREATE TABLE user (
- id INT NOT NULL AUTO_INCREMENT UNIQUE,
-  first name VARCHAR(30) NOT NULL , 
- last name VARCHAR(30) NOT NULL ,  
-password VARCHAR(20) NOT NULL , 
- email VARCHAR(20) NOT NULL , 
- address VARCHAR(50) NOT NULL ,
-  phone VARCHAR(20) NOT NULL , 
-   PRIMARY KEY  (id)
-) 
-
-CREATE TABLE product ( 
-p_id INT NOT NULL AUTO_INCREMENT UNIQUE, 
- p_price INT NOT NULL ,
-  p_quantity INT NOT NULL ,
-  p_description MEDIUMTEXT NOT NULL ,
-  p_image VARCHAR(50) ,
-    PRIMARY KEY  (p_id)
-)
-
-CREATE TABLE order (
- o_id INT NOT NULL AUTO_INCREMENT UNIQUE,
-  id INT NOT NULL , 
- p_id INT NOT NULL ,
-  quantity INT NOT NULL ,
-    PRIMARY KEY  (o_id) ,
-FOREIGN KEY (id) REFERENCES Persons(id),
-FOREIGN KEY (p_id) REFERENCES Persons(p_id)
-) 
\ No newline at end of file
+  id INT NOT NULL AUTO_INCREMENT UNIQUE,
+  first name VARCHAR(30) NOT NULL,
+  last name VARCHAR(30) NOT NULL,
+  password VARCHAR(20) NOT NULL,
+  email VARCHAR(20) NOT NULL,
+  address VARCHAR(50) NOT NULL,
+  phone VARCHAR(20) NOT NULL,
+  PRIMARY KEY (id)
+) CREATE TABLE product (
+  p_id INT NOT NULL AUTO_INCREMENT UNIQUE,
+  p_price INT NOT NULL,
+  p_quantity INT NOT NULL,
+  p_description MEDIUMTEXT NOT NULL,
+  p_image VARCHAR(50),
+  created_at timestamp NOT NULL DEFAULT current_timestamp(),
+  updated_at timestamp NULL DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+  PRIMARY KEY (p_id)
+) CREATE TABLE order (
+  o_id INT NOT NULL AUTO_INCREMENT UNIQUE,
+  id INT NOT NULL,
+  p_id INT NOT NULL,
+  quantity INT NOT NULL,
+  PRIMARY KEY (o_id),
+  FOREIGN KEY (id) REFERENCES Persons(id),
+  FOREIGN KEY (p_id) REFERENCES Persons(p_id)
+)
```

Comment:
this file no longer needed

Reply:
should i create a new one ?
i think we should not because we have migration folder .